dist/components/badge/_badge.scss

```diff
@@ -0,0 +1,34 @@
+// Badge Component
+// ===
+//
+// A badge icon with content. Often used to show a the quantity of items in a
+// cart and paired visually with the cart icon in the header.
+
+
+// Component Variables
+// ---
+//
+
+$badge__vertical-padding: 0.25em !default;
+$badge__horizontal-padding: 0.7em !default;
+$badge__background-color: #ccc !default;
+$badge__border-radius: 4em !default;
+$badge__text-color: white !default;
+$badge__font-size: 0.75em !default;
+$badge__font-weight: 600;
```

Comment:
Missing a `!default`, but I think this variable can probably be removed anyway; see my general comments below.